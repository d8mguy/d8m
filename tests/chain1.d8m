
val foo = \imp(lst : list(integer)) {
    val interm1 = lst.[this % 2 == 1].{this*this}
    val sum1 = sigma(interm1)
    interm1 + lst.[this % 2 == 0].{this * sum1}
}
val lst0 = [1,5,3,77,34,2,12]
foo(lst0)
