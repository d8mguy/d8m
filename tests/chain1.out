[1, 25, 9, 5929, 202776, 11928, 71568]
